src/monodroid/jni/Android.mk

```diff
@@ -1,5 +1,8 @@
 LOCAL_PATH := $(call my-dir)
 
+JAVA_INTEROP_INCLUDES=../../external/Java.Interop
```

Comment:
I'm still confused as to *how this works*: relative to `src/monodroid/jni`, `../../external` doesn't exist.

This strongly suggests that `$(CWD)` isn't necessarily what we think it is. This further suggests that we should *instead* use a `$(LOCAL_PATH)`-relative value, *or* `$(JAVA_INTEROP_PATH)` should be defined at the `ndk-build` invocation, [just like `MONO_PATH` is](https://github.com/xamarin/xamarin-android/blob/5797bea8f031dd3d71812d02b275b795dd46fef1/src/monodroid/monodroid.targets#L51):

```xml
      <_NdkBuildArgs>CONFIGURATION=$(Configuration) SGEN_BRIDGE_VERSION=$(MonoSgenBridgeVersion) JAVA_INTEROP_PATH="$(JavaInteropFullPath)" MONO_PATH="$(MonoSourceFullPath)"</_NdkBuildArgs>
```

Reply:
OK, let use the `$(LOCAL_PATH)` then. It is set to `jni` during the build.